app/src/org/commcare/preferences/AdvancedActionsPreferences.java

```diff
@@ -71,26 +78,38 @@
         keyToTitleMap.put(CLEAR_SAVED_SESSION, "menu.clear.saved.session");
     }
 
+    @NonNull
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+    protected String getTitle() {
+        return Localization.get("settings.advanced.title");
+    }
 
-        addPreferencesFromResource(R.xml.advanced_actions);
+    @Nullable
```

Comment:
Yeah. Android Studio does add all annotations from base function automatically and gives a warning if we omit them. So it definitely seems like annotations doesn't get inherited automatically in Android Studio.